src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -748,6 +750,12 @@ object Parser {
     lazy val cap : Parser[PCapacity] =
       "cap" ~> ("(" ~> expression <~ ")") ^^ PCapacity
 
+    lazy val keys : Parser[PMapKeys] =
+      "keySet" ~> ("(" ~> expression <~ ")") ^^ PMapKeys
+
+    lazy val values : Parser[PMapValues] =
+      "valueSet" ~> ("(" ~> expression <~ ")") ^^ PMapValues
+
```

Comment:
Why did you pick these names and not the names used by e.g. Viper?

Reply:
I was trying to avoid overloading `range`. However, I believe that can be changed to the `domain` and `range` from Viper easily